index.html

```diff
@@ -0,0 +1,116 @@
+<!doctype html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=0.1">
+	<title>Ellie Robinson - Developer</title>
+	<link href="normalise.css" type="text/css" rel="stylesheet"> <!-- this makes sure all browsers start with the same style -->
+	<link href="style.css" type="text/css" rel="stylesheet">
+	<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap" rel="stylesheet">
+	<link href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@200;300&display=swap" rel="stylesheet"> 
+     <link href="https://fonts.googleapis.com/css2?family=Noto+Sans&display=swap" rel="stylesheet">
+	<script src="https://kit.fontawesome.com/d8f8b9f853.js" crossorigin="anonymous"></script>
+</head>
+<body>
+	<section class="navigationBar">
+		<div>
+			<img src="./externalFiles/Ellie-Robinson2.png">
+		</div>
+		<nav>
+			<a href="#AboutMe">About Me</a>
+			<a href="#Portfolio">Portfolio</a>
+			<a href="#Contact">Contact</a>
+		</nav>
+	</section>
+	<header class="welcome">
+		<div class="intro">
+			<h1>Welcome!</h1>
+			<p>I'm currently training to be a <br>full stack developer at Mayden Academy.<br> Feel free to have a look around.</p>
```

Comment:
Don't use <br> tag - not great for accessibility. You can put the two sentences in separate <p> tags and set a max width for the text.